cylc/flow/suite_files.py

```diff
@@ -733,122 +624,10 @@ def get_suite_title(reg):
     return title
 
 
-@lru_cache()
-def _is_local_auth_ok(reg, owner, host):
-    """Return True if it is OK to use local passphrase file.
-
-    Use values in ~/cylc-run/REG/.service/contact to make a judgement.
-    """
-    if is_remote(host, owner):
-        fname = os.path.join(
-            get_suite_srv_dir(reg), SuiteFiles.Service.CONTACT)
-        data = {}
-        try:
-            for line in open(fname):
-                key, value = (
-                    [item.strip() for item in line.split("=", 1)])
-                data[key] = value
-        except (IOError, ValueError):
-            # No contact file
-            return False
-        else:
-            # Contact file exists, check values match
-            if owner is None:
-                owner = get_user()
-            if host is None:
-                host = get_host()
-            host_value = data.get(ContactFileFields.HOST, "")
-            return (
-                reg == data.get(ContactFileFields.NAME) and
-                owner == data.get(ContactFileFields.OWNER) and
```

Comment:
I guess this breaks the `--owner` option, I think we can do this as editing your SSH config is a much nicer way to SSH to a host as another user.

Did we agree to get rid of it, probably but I can't remember.

Reply:
I am really not sure about this one, it was left hanging. Happy to continue stripping out the `--owner` code if people would rather have this set in SSH config. Perhaps useful to have in Cylc if it is a feature used on a regular basis, although I am not sure about its widespread use.

Reply:
I think we can get rid of `--owner` now @hjoliver

Reply:
(I think we agreed this in the last video call)

Reply:
Yes agreed!